### DISCUSSION TITLE
监听主机能不能可以设置成域名呢？

### DISCUSSION THREAD

**Discussion by a85559775** [messense/aliyundrive-webdav#406] [Q&A]
### 问题描述

监听主机能不能可以设置成域名呢？？我填写路由器获得的公网ipv6，外网也可以访问的，能不能可以设置成填写域名呢？这样不用每次公网ipv6变化，我都要去手动修改。

### 重现步骤

1.
2.
3.

### 版本

1.31

### 运行平台

x86软路由

### 日志

--

**Comment by messense** [comment]
`::`
